src/tabbar.cpp

```diff
@@ -26,42 +26,66 @@ TabBar::TabBar(QWidget *parent) :
     auto app = KiwixApp::instance();
     connect(app->getAction(KiwixApp::NewTabAction), &QAction::triggered,
             this, [=]() {
-                this->createNewTab(true);
```

Comment:
@mgautierfr  Do we have any code convention on Kiwix Desktop project? If not, I suggest using Qt's convention: https://wiki.qt.io/Coding_Conventions#Lambdas